src/systemjs.config.js

```diff
@@ -1,52 +1,43 @@
-(function(global) {
-
-  // map tells the System loader where to look for things
-  var map = {
-      'app': 'app',
-      'rxjs': 'assets/vendor/rxjs',
-      'angular2-in-memory-web-api': 'assets/vendor/angular2-in-memory-web-api',
-      '@angular': 'assets/vendor/@angular'
-  };
-  // packages tells the System loader how to load when no filename and/or no extension
-  var packages = {
-      'app': {
-          main: 'boot.js',
-          defaultExtension: 'js'
+/**
+ * System configuration for Angular samples
+ * Adjust as necessary for your application needs.
+ */
+(function (global) {
+  System.config({
+    paths: {
+      // paths serve as alias
+      'npm:': '../node_modules/'
```

Comment:
The issue with this is that the intention is to be able to build a distributable application, and doing this would require ALL node dependencies to be copied into the build. Maybe look at changing this to what it was before, and copying over dependencies into dist with Gulp?